### PR TITLE
Missing indexdts

### PR DESCRIPTION
**Pull request by xerona**

### CODE REVIEW

**Comment by xerona** [comment]
@caffeinelogic @craigmichaelmartin @akersinformz

File: angular-cli.json [hunk 1]
```diff
@@ -23,9 +23,9 @@
       ],
       "scripts": [],
       "environments": {
-        "source": "environments/environment.ts",
-        "dev": "environments/environment.ts",
-        "prod": "environments/environment.prod.ts"
+        "source": "../example/environments/environment.ts",
+        "dev": "../example/environments/environment.ts",
+        "prod": "../example/environments/environment.prod.ts"
```
Comment (caffeinelogic) [inline angular-cli.json]: Just curious, did these cause issues?

File: dist/login/login.component.html [hunk 2]
```diff
@@ -1,38 +1,48 @@
...
+      <!-- password input -->
       <div class="form-group">
-        <label for="password">PASSWORD</label>
-        <input
-          class="_m0"
-          name="password"
-          type="password"
-          required
-          [ngModel]="user.password">
+        <label class="u-type-s u-bold color-mid-gray _pbxs" for="password">PASSWORD</label>
```
Comment (caffeinelogic) [inline dist/login/login.component.html]: lets add the `for` attribute and the corresponding `id` to the input.

File: dist/login/login.component.html [hunk 3]
```diff
@@ -1,38 +1,48 @@
...
+        <input class="_m0" name="password" type="password" required [ngModel]="user.password">
```
Comment (caffeinelogic) [inline dist/login/login.component.html]: Lets add `autocomplete="off"` to all our inputs here, best practice security wise.

File: dist/login/login.component.html [hunk 4]
```diff
@@ -1,38 +1,48 @@
...
+
+<!-- begin login form -->
+    <form class="_ptxl"
+          #loginForm="ngForm"
+          (ngSubmit)="login(loginForm)">
+
+      <!-- username input -->
+      <div class="form-group _pbxl">
+        <label class="u-type-s u-bold color-mid-gray _pbxs" for="username">USER NAME</label>
+        <input class="_m0" name="username" required [ngModel]="user.username">
```
Comment (caffeinelogic) [inline dist/login/login.component.html]: Lets add autocomplete="off" to all our inputs here, best practice security wise.

File: dist/login/login.component.html [hunk 5]
```diff
@@ -1,38 +1,48 @@
...
+    <img class="Login-title" [src]="titlePath" />
+
+<!-- begin login form -->
+    <form class="_ptxl"
+          #loginForm="ngForm"
+          (ngSubmit)="login(loginForm)">
+
+      <!-- username input -->
+      <div class="form-group _pbxl">
+        <label class="u-type-s u-bold color-mid-gray _pbxs" for="username">USER NAME</label>
```
Comment (caffeinelogic) [inline dist/login/login.component.html]: lets add the for attribute and the corresponding id to the input.

File: angular-cli.json (hunk 1 above)
Comment (xerona) [inline angular-cli.json]: It didn't till last night. I have no idea what happened, but that is the correct path.

File: src/login/login.component.html [hunk 6]
```diff
@@ -1,38 +1,48 @@
...
+
+<!-- begin login form -->
+    <form class="_ptxl"
+          #loginForm="ngForm"
+          (ngSubmit)="login(loginForm)">
+
+      <!-- username input -->
+      <div class="form-group _pbxl">
+        <label class="u-type-s u-bold color-mid-gray _pbxs" for="username">USER NAME</label>
+        <input class="_m0" name="username" required [ngModel]="user.username">
```
Comment (caffeinelogic) [inline src/login/login.component.html]: Lets add `autocomplete="off"` to all our inputs here, best practice security wise.

You're missing the `id` attribute that goes with the `for` for the label.

File: src/login/login.component.html [hunk 7]
```diff
@@ -1,38 +1,48 @@
...
       <div class="form-group">
-        <label for="password">PASSWORD</label>
-        <input
-          class="_m0"
-          name="password"
-          type="password"
-          required
-          [ngModel]="user.password">
+        <label class="u-type-s u-bold color-mid-gray _pbxs" for="password">PASSWORD</label>
+        <input class="_m0" name="password" type="password" required [ngModel]="user.password">
```
Comment (caffeinelogic) [inline src/login/login.component.html]: Lets add `autocomplete="off"` to all our inputs here, best practice security wise.

You're missing the `id` attribute that goes with the `for` for the label.

**Pull request merged**